review/fileToReview.js

```diff
@@ -0,0 +1,73 @@
+
+var domain = "bank.local.fr"
```

Comment:
Utilisation d'une constante si cette variable ne change pas :
```suggestion
const DOMAIN = "bank.local.fr"
```